tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/Properties/InterceptedProjectProperties/ActiveLaunchProfileValueProvidersTests.cs

```diff
@@ -56,9 +56,9 @@ public async Task ExecutablePath_OnGetEvaluatedPropertyValueAsync_GetsExecutable
 
             var project = UnconfiguredProjectFactory.Create();
             var threadingService = IProjectThreadingServiceFactory.Create();
-            var launchProfileProvider = new ExecutablePathValueProvider(project, settingsProvider, threadingService);
+            var launchProfileProvider = new ActiveLaunchProfileCommonValueProvider(project, settingsProvider, threadingService);
 
-            var actualValue = await launchProfileProvider.OnGetEvaluatedPropertyValueAsync(string.Empty, string.Empty, Mock.Of<IProjectProperties>());
+            var actualValue = await launchProfileProvider.OnGetEvaluatedPropertyValueAsync("ExecutablePath", string.Empty, Mock.Of<IProjectProperties>());
```

Comment:
>"ExecutablePath" [](start = 91, length = 16)

Would it be better to use the constants from ActiveLaunchProfileCommonValueProvider instead of all the inlined strings?

Reply:
Done.